lib/kms.js

```diff
@@ -37,12 +59,19 @@ export async function createKeystore({
     keystoreConfig.referenceId = referenceId;
   }
   const {httpsAgent} = brHttpsAgent;
-  return KmsClient.createKeystore({
-    url: `${bedrock.config.tokenizer.kms.baseUrl}/keystores`,
-    config: keystoreConfig,
-    invocationSigner,
-    httpsAgent
-  });
+  let result;
+  try {
+    result = await KmsClient.createKeystore({
+      url: `${bedrock.config.tokenizer.kms.baseUrl}/keystores`,
+      config: keystoreConfig,
+      invocationSigner,
+      httpsAgent
+    });
+  } catch(cause) {
+    throw new BedrockError('Error creating keystore for tokenizer.',
+      'WebKMSClientError', null, cause);
+  }
```

Comment:
Indent on param, indent them all.

Second, please do not create error names like this `WebKMSClientError`.  Whenever possible use something from this list: https://developer.mozilla.org/en-US/docs/Web/API/DOMException

Since the error is not being inspected to try to determine what is happening here, I think the appropriate option is `UnknownError`.

Also, `null` for the error details is unusual.  It usually contains.  Without this data, if this error floats up the the `bedrock-express` default error handler, it will be surfaced as an opaque "Internal Server Error".
```
{httpStatusCode: 400, public: true}
```

Lastly, I wonder if a `logger.error` call is in order for this.  Otherwise, there can be no assurance that an error like this will be recorded in the logs.

Reply:
@mattcollier fixed indentation, name, and details.

> Lastly, I wonder if a logger.error call is in order for this. Otherwise, there can be no assurance that an error like this will be recorded in the logs.

What's the preferred convention -- should `bedrock-tokenizer` fire up its own logger, or should one be passed down from the parent app?

Reply:
@dmitrizagidulin, `bedrock-tokenizer` should fire up its own logger as a "child" logger of the app. An example:

https://github.com/digitalbazaar/bedrock-kms-http/blob/main/lib/logger.js

Reply:
@dlongley - k, added.